tests/test_ext_napoleon_docstring.py

```diff
@@ -1118,7 +1130,7 @@ def test_noindex(self):
 .. method:: func(i, j)
    :noindex:
 
-   
+
```

Comment:
I guess this change causes an error in CI. Is this an intended change? Please check it.

Reply:
Oh yeah must have been the line trimming

Reply:
```suggestion
   
```